include/boost/gil/algorithm.hpp

```diff
@@ -777,13 +814,19 @@ void uninitialized_copy_aux(It1 first1, It1 last1, It2 first2, std::false_type)
 template <typename View1, typename View2>
 void uninitialized_copy_pixels(View1 const& view1, View2 const& view2)
 {
-    using is_planar = std::integral_constant<bool, is_planar<View1>::value && is_planar<View2>::value>;
+    using copy_planarity_condition = detail::copy_planarity_condition;
+    using vxv_planarity = std::integral_constant<copy_planarity_condition,
```

Comment:
What does the `vxv` stand for?

Reply:
my bad, that's a placeholder name that I should've removed. It stood for 'value x value". I'll replace it with "copy_planarity_condition_type"